sqlathanor/declarative/base_model.py

```diff
@@ -265,17 +265,17 @@ def _get_deserialized_value(cls,
                 # pylint: disable=W0212
                 if hasattr(resolved_class, 'new_from_json') and format == 'json' and isinstance(value, dict):
                     as_json = json.dumps(value)
-                    return_value = resolved_class.new_from_json(as_json, **kwargs)
+                    return_value = resolved_class.new_from_json(as_json, **kwargs, config_set=config_set)
                 elif hasattr(resolved_class, 'new_from_yaml') and format == 'yaml' and isinstance(value, dict):
                     as_yaml = yaml.dump(value)
-                    return_value = resolved_class.new_from_yaml(value, **kwargs)
+                    return_value = resolved_class.new_from_yaml(value, **kwargs, config_set=config_set)
```

Comment:
This syntax unfortunately is not backwards compatible with earlier versions of Python supported by SQLAthanor. The issue that ``**kwargs`` must be the last keyword argument in the function call, so this statement would have to be restated as:

``resolved_class.new_from_json(as_json, config_set = config_set, **kwargs)``